CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -192,11 +195,18 @@ protected function getTemplateUpdates() {
           ['name' => 'petition_sign', 'type' => 'subject'],
           ['name' => 'petition_sign', 'type' => 'text'],
           ['name' => 'pledge_acknowledge', 'type' => 'subject'],
-          ['name' => 'pledge_acknowledge', 'type' => 'subject'],
+          ['name' => 'pledge_acknowledge', 'type' => 'html'],
```

Comment:
@eileenmcnaughton given i'm not sure if it was meant to be a dupe or not here or whatever decided to be on the safe side and upgrade all 3 template aspects

Reply:
I doubt it was meant to be a dupe